lib/firebase/src/user.dart

```diff
@@ -177,6 +201,9 @@ sealed class ThcUser {
         if (email != null) 'email': email,
         if (!registered) 'registered': false,
         if (profilePictureUrl != null) 'profilePictureUrl': profilePictureUrl,
+        if (isAudioEnabled != isAudioEnabled) 'isAudioEnabled': isAudioEnabled,
+        if (isVideoEnabled != isVideoEnabled) 'isVideoEnabled': isVideoEnabled,
+        if (view != view) 'view' : view,
```

Comment:
A couple of questions here:

1. It seems as though `if (x != x)` will always return false, do you know what the intended behavior is here?
2. I noticed that `view` has a the `Widget?` type, and I'm not exactly sure how we're intending to represent it in JSON format.
(I also haven't seen anywhere that sets a value for `view`.)